### PR TITLE
fixed virtual paging

### PR DESCRIPTION
**Pull request by bensgroi**
Removed the external paging from the demo. Issue #33

### CODE REVIEW

File: demos/virtual-paging.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="apple-mobile-web-app-capable" content="yes" />
-    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
-    <meta name="viewport" content="width = device-width, minimal-ui, initial-scale = 1, user-scalable = no" />
-    <meta name="apple-mobile-web-app-title" content="Datagrid">
-    <title>Datagrid - Paging</title>
-    <base href="/">
-    <style>
+
+<head>
```
Comment (jonshaffer) [inline demos/virtual-paging.html]: it looks like it did the tabbing thing again, would it help you if i implemented html lint/hint to the project?

File: demos/virtual-paging.html [hunk 2]
```diff
@@ -74,37 +71,16 @@
               { name: "Gender", prop: "gender", sortable: false },
               { name: "Company", prop: "company", sortable: false }
             ],
-            columnMode: 'force',
-            paging: {
-              externalPaging: true,
-              loadingIndicator: true
-            }
+            columnMode: 'force'
```
Comment (jonshaffer) [inline demos/virtual-paging.html]: `scrollbarV: true,` ?

Comment (bensgroi) [inline demos/virtual-paging.html]: It defaults to true

Comment (jonshaffer) [inline demos/virtual-paging.html]: :o)

**Pull request closed without merging**